### PR TITLE
Change examples structure

### LINKED ISSUE #410: Message object examples facet has a wrong type
Issue part of #397

It’s an array of Maps, it should be only a Map. Message trait objects have the same problem.
https://github.com/asyncapi/asyncapi/blob/master/versions/2.0.0/asyncapi.md#message-object
https://github.com/asyncapi/asyncapi/blob/master/versions/2.0.0/asyncapi.md#message-trait-object

### PR DESCRIPTION
**Pull request by BlockLucas**
Now examples facet for Message & Message Trait objects are an array of any
Fixes #410

### CODE REVIEW

File: versions/2.0.0/asyncapi.md [hunk 1]
```diff
@@ -1003,7 +1003,7 @@ Field Name | Type | Description
 <a name="messageObjectTags"></a>tags | [Tags Object](#tagsObject) | A list of tags for API documentation control. Tags can be used for logical grouping of messages.
 <a name="messageObjectExternalDocs"></a>externalDocs | [External Documentation Object](#externalDocumentationObject) | Additional external documentation for this message.
 <a name="messageObjectBindings"></a>bindings | [Message Bindings Object](#messageBindingsObject) | A map where the keys describe the name of the protocol and the values describe protocol-specific definitions for the message.
-<a name="messageObjectExamples"></a>examples | [Map[`string`, `any`]] | An array with examples of valid message objects.
+<a name="messageObjectExamples"></a>examples | [`any`] | An array with examples of valid message objects. Whatever "any" is, it has to match the type defined on payload.
```
Comment (fmvilas) [inline versions/2.0.0/asyncapi.md:1006]:
```suggestion
<a name="messageObjectExamples"></a>examples | [`any`] | An array with examples of valid message objects. Whatever "any" is, it MUST match the type defined on the payload. 
```

File: versions/2.0.0/asyncapi.md [hunk 2]
```diff
@@ -1168,7 +1168,7 @@ Field Name | Type | Description
 <a name="messageTraitObjectTags"></a>tags | [Tags Object](#tagsObject) | A list of tags for API documentation control. Tags can be used for logical grouping of messages.
 <a name="messageTraitObjectExternalDocs"></a>externalDocs | [External Documentation Object](#externalDocumentationObject) | Additional external documentation for this message.
 <a name="messageTraitObjectBindings"></a>bindings | [Message Bindings Object](#messageBindingsObject) | A map where the keys describe the name of the protocol and the values describe protocol-specific definitions for the message.
-<a name="messageTraitObjectExamples"></a>examples | [Map[`string`, `any`]] | An array with examples of valid message objects.
+<a name="messageTraitObjectExamples"></a>examples | [`any`] | An array with examples of valid message objects. Whatever "any" is, it has to match the type defined on payload.
```
Comment (fmvilas) [inline versions/2.0.0/asyncapi.md:1171]:
```suggestion
<a name="messageTraitObjectExamples"></a>examples | [`any`] | An array with examples of valid message objects. Whatever "any" is, it MUST match the type defined on the payload. 
```

**Review by fmvilas — CHANGES_REQUESTED** [review]
Left some minor tweaks to convey with spec language. Thanks a lot, @BlockLucas!

**Comment by BlockLucas** [comment]
Thanks, @fmvilas!
Suggestions applied

**Comment by fmvilas** [comment]
Awesome! Thanks, @BlockLucas!

**Pull request merged**